windup/windup-operator
# windup-3016 Executor pods not affecting web console pods

https://issues.redhat.com/browse/WINDUP-3016

Instructions to test :

    1. oc login -u kubeadmin -p { pwd} --server={your environment}
    2. ./create-catalog.sh -u {quay user} -p { quay pwd } -m 0.0.3 -b 1
    3. on ocp ui
    3.1. create "mta" project
    3.2. install operator 0.0.3 from test-catalog on "mta" project
    3.3. create mta instance (specify 3 for desired-executor-replicas )
    
Expected results

     only 1 pod for "mta" deployment and 3 pods for "mta-executor" deployment

  4. got to mta operator installed
  5. click on it and go to "Migration Toolkit for Applications" tab
  6. click on the mta instance
  7. click on the yaml
  8. modify the value for "executor_desired_replicas" to 2  ( you can click Ctrl+F to search in the yaml)
  9. go to Workloads ( left menu)
 10. check for the Deployments in the "mta" project
 
Expected results
    
     only 1 pod for "mta" deployment and 2 pods for "mta-executor" deployment
